### PR TITLE
Tracking of bad boxes

### PR DESCRIPTION
**Pull request by pantheras**
Spits out warnings for bad boxes, required when writing academic papers to notify of potential formatting issues

### CODE REVIEW

**Comment by fabrice102** [comment]
I strongly support this change too.

File: parseTeXlog.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def advance_iterator(it):
 
 def debug(s):
 	if print_debug:
-		print ("parseTeXlog: " + s.encode('UTF-8')) # I think the ST2 console wants this
+		print ("parseTeXlog: " + s) #s.encode('UTF-8')) # I think the ST2 console wants this
```
Comment (ig0774) [inline parseTeXlog.py]: Is this change necessary for these changes to work? It looks as though it may be harmful e.g. If utf-8 characters somehow make it into one of the output messages.

**Comment by aleth** [comment]
+1

**Comment by ig0774** [comment]
This change has been merged, though by default, displaying bad boxes is disabled.
